paymentsheet/src/main/java/com/stripe/android/link/LinkController.kt

```diff
@@ -61,7 +62,37 @@ class LinkController @Inject internal constructor(
     }
 
     /**
-     * [CRYPTO ON-RAMP ONLY] Authenticate with Link.
```

Comment:
👀 I did a bit of housecleaning and moved crypto stuff below. Definitely unnecessary at this point, but I did it while trying to think through the API and forgot to undo. Also cleaned up comments to be "onramp" instead of "on-ramp"